libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -1,58 +1,71 @@
-import PauseCircleFilledIcon from '@mui/icons-material/PauseCircleFilled';
-import PlayCircleFilledIcon from '@mui/icons-material/PlayCircleFilled';
-import { useEffect, useState } from 'react';
-import styles from './MediaPlayer.module.scss';
+import { styled, useTheme } from '@mui/material';
+import { useEffect, useRef } from 'react';
 
 export interface MediaPlayerProps {
     audioUrl: string;
     listenMessage?: string;
 }
 
-const DEFAULT_LISTEN_MESSAGE = 'Listen Live!';
+export function MediaPlayer({ audioUrl }: MediaPlayerProps) {
+    const audioRef = useRef<HTMLAudioElement | null>(null);
 
-export function MediaPlayer({ audioUrl, listenMessage }: MediaPlayerProps) {
-    const [audio, setAudio] = useState<HTMLAudioElement | null>(null);
+    useEffect(() => {
+        if (audioRef.current !== null) {
+            audioRef.current.src = audioUrl;
+        }
+    }, [audioUrl]);
 
-    const [canPlayThrough, setCanPlayThrough] = useState(false);
+    const {
+        palette: {
+            secondary: { main },
```

Comment:
If this is in a library, how does this interact with the MUI theme for the frontend?